### PR TITLE
Redirect A/B test variant page to home

### PR DESCRIPTION
**Pull request by gustaveen**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2762** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2762" target="_blank">Graphite</a> 
    * **PR #2764** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2764" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2764?utm_source=stack-comment).

File: apps/store/next.config.js [hunk 1]
```diff
@@ -109,6 +109,16 @@ module.exports = withBundleAnalyzer({
         permanent: true,
         locale: false,
       })),
+      ...(process.env.NODE_ENV === 'production'
+        ? [
+            {
+              source: `/se/start`,
```
Comment (robinandeer) [inline apps/store/next.config.js:115]: Can we read these from the AB test object?

Would be hard to keep them in sync otherwise, no?

Comment (gustaveen) [reply apps/store/next.config.js:115]: Hm, we're not able to import the AB test object into the config though right? @robinandeer

Comment (robinandeer) [reply apps/store/next.config.js:115]: Hmm no I don't think so -- we would probably have to move the config object outside JS

In some place that we can access in both places

Maybe in "package.json" or some custom JSON file?

Comment (gustaveen) [reply apps/store/next.config.js:115]: Gotcha, or could it better be done where we handle the experiment middleware? Feel free to discard this solution 🙌

**Review by guilhermespopolin — APPROVED** [review]
Nice that config is localised in single file. Well done :)

**Pull request merged**